src/flir_adk_ethernet/EthernetCamera.cpp

```diff
@@ -92,6 +105,12 @@ bool EthernetCamera::findMatchingCamera(CameraListWrapper camList, const unsigne
         CameraWrapper cam = camList.GetByIndex(i);
         INodeMap &nodeMapTLDevice = cam.GetTLDeviceNodeMap();
 
+        CBooleanPtr wrongSubnet = nodeMapTLDevice.GetNode("GevDeviceIsWrongSubnet");
+        if (wrongSubnet->GetValue())
+        {
```

Comment:
fix bracing style (open on same line)
also, check to see if the rest of this code uses braces for single line statements.

maybe we should add a logging statement to the effect of "skipping invalid subnet: <subnet>", anyway.

Reply:
rest of the package uses braces for single line statements